app/controllers/idv/document_capture_controller.rb

```diff
@@ -9,6 +9,7 @@ class DocumentCaptureController < ApplicationController
 
     before_action :confirm_hybrid_handoff_complete
     before_action :confirm_document_capture_needed
+    before_action :cancel_establishing_in_person_enrollments
```

Comment:
I would expect the cancellation to happen when the user clicks cancel and start over, rather than on viewing the document capture page. We're planning to make it much easier for users to click the back button, and we wouldn't want viewing a page to have unexpected side effects.

Reply:
That does happen now, but it's possible to get back to document capture without clicking 'cancel' (see steps to reproduce in the testing plan).

Reply:
I can't think of any circumstance in which we'd want to have an 'establishing' enrollment when the user loads the doc capture page, no matter how they got there.